libraries/Microsoft.Bot.Builder.Ai/SmartTalk.cs

```diff
@@ -0,0 +1,92 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Net.Http;
+using Newtonsoft.Json;
+using System.Threading.Tasks;
+using System.Linq;
+
+namespace Microsoft.Bot.Builder.Ai
+{
+    public class SmartTalk
+    {
+        public const string smartTalkServiceEndPoint = "https://smarttalkmanagementservice.azure-api.net/smarttalk/api/SmartTalk/GetResponse";
+        public const string APIManagementHeader = "Ocp-Apim-Subscription-Key";
+        public const string JsonMimeType = "application/json";
+
+        private static HttpClient g_httpClient = new HttpClient();
+        private readonly HttpClient _httpClient;
```

Comment:
It seems really weird to have both a static and an instance version of this. I get the motivation (these things are expensive), but it's... strange. If you're worried about it, just require it be passed in.